cabby/data/wikidata/query.py

```diff
@@ -0,0 +1,192 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Library to support Wikidata areal queries.'''
+
+import sys
+from SPARQLWrapper import SPARQLWrapper, JSON
+from typing import Dict, Tuple, Sequence, Text
+
+
+def get_wikidata(map_name: Text) -> Dict:
```

Comment:
Dict of what? E.g. should be typed as in Dict[Text, int]

Reply:
done